datanode/gateway/graphql/schema.graphql

```diff
@@ -474,11 +475,15 @@ type Query {
     "Optional Pagination"
     pagination: Pagination): MarketDataConnection
 
+  "get historical balances for an account or specific asset within the given date range."
```

Comment:
```suggestion
  "Get historical balances for an account or specific asset within the given date range."
```

Reply:
There appears to be a lot of these docstrings where the first word in the sentence is not capitalised. I'll check and make sure the others are updated too.